build/build-bundle.js

```diff
@@ -64,7 +64,14 @@ async function browserifyFile(entryPath, distPath) {
     .ignore('raven')
     .ignore('mkdirp')
     .ignore('rimraf')
-    .ignore('pako/lib/zlib/inflate.js');
+    .ignore('pako/lib/zlib/inflate.js')
+    .ignore('file') // required by jsonlint-mod
+    .ignore('system'); // required by jsonlint-mod
+
+  // There is no way to add './doug-json-parse' to ignored packages via public API
+  // w/o browserify resolving the path into an absolute path
+  // @ts-ignore
+  bundle._ignore.push('./doug-json-parse');
```

Comment:
this feels like a recipe for struggles on next browserify update 😬 

maybe it's time to investigate moving to something better maintained/more flexible post 5.0

Reply:
still a bit concerned about this but I think whatever we end up doing will be hacky in some way 🤷‍♂️

Reply:
should we add a test that the bundle output does not contain some code in `douge-json-parse`? so we remember to fix this when it break.

Reply:
> should we add a test that the bundle output does not contain some code in douge-json-parse? so we remember to fix this when it break.

the build throws if we forget this, so it will definitely fail the tests :)

The issue is that `jsonlint-mod`'s bundled js has a reference to this file, which isn't included in the module. Browserify throws when it tries to resolve it.